### DISCUSSION TITLE
[Nuxt-bridge / VS Code] Can't import from #import

### DISCUSSION THREAD

**Discussion by Lucky-Strike1695** [nuxt/nuxt#16086] [Questions]
Hi,

I'm migrating from Nuxt 2.x to Nuxt-Bridge. I need to stay on 2.x as long as nuxt-auth-next isn't compatible with the 3.x branch.
Everything seems to be working fine exept imports in my `/server/api`.

The following code gives `defineEventHandler()`, `useRuntimeConfig()` and `readBody()` are undefined.
```javascript
/* /server/api/login.post.ts */
export default defineEventHandler(async (event) => {
  const config = useRuntimeConfig()
  const user = await readBody(event)
  // ...
})
```

Note that the code _does_ work no matter what VS Code says.

I can fix `useRuntimeConfig()` and `readBody()` errors by adding `import { defineEventHandler, readBody } from 'h3' ` to manually and explicitly importing it.
Morever, for `defineEventHandler()`, I can't find any working way. I tried `import { useRuntimeConfig } from '#imports'` (and '#app') but it says that #imports cannot be located (tried with several modules in .nuxt : all fail).
Also, `import { useRuntimeConfig } from '~/.nuxt/imports'` gets rid of the error is ESLint but nuxi crashes with  : 
`ERROR  [worker reload] [worker init] Only URLs with a scheme in: file, data are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs.`
Using `'../../.nuxt/imports.mjs'` gives  `ERROR  [worker reload] [worker init] Package import specifier "#head" is not defined in package`

I'm running VS Code with Volar. I tried Take over mode and normal, without any change. 

Thanks in advance for your advices.

**Comment by mubaidr** [comment]
I think I am having the similar issue here, imports from `#import`, `#head`, `#app` are not resolved. 

More details: https://github.com/nuxt/framework/issues/9466
